locales/zh/filters.json

```diff
@@ -159,4 +159,8 @@
 		"hostlist.42.name": "ShadowWhisperer's Malware List",
 		"hostlist.42.description": "恶意网站、PUP、恶意软件、浏览器劫持程序、钓鱼网站"
 	}
```

Comment:
The comma is forgotten here.

And please add also English localization.

Reply:
It has been modified as required, please review, thank you!